lib/far_mar.rb

```diff
@@ -0,0 +1,6 @@
+require "./lib/far_mar/market"
+require "./lib/far_mar/vendor"
+require "./lib/far_mar/product"
+require "./lib/far_mar/sale"
+require "csv"
```

Comment:
This `require` line and the one following it should come first, as each of the FarMar files will make use of them. If you require them afterwards then the code in those files won't be able to access the CSV or Datetime classes.